### PR TITLE
Add leap exercise

### PR DESCRIPTION
**Pull request by lpil**

### CODE REVIEW

**Comment by paf31** [comment]
👍 Looks good!

**Comment by paf31** [comment]
@kytrinyx Once Travis is updated (#9), shall I just merge these if the tests pass and the code looks right?

**Comment by kytrinyx** [comment]
@paf31 Yeah, totally!

**Comment by lpil** [comment]
Rebased, tests are green :)

File: exercises/hello-world/src/HelloWorld.purs [hunk 1]
```diff
@@ -1,6 +1 @@
 module HelloWorld where
-
```
Comment (lpil) [inline exercises/hello-world/src/HelloWorld.purs]: I looked at the tracks for Haskell and Elm and saw that they didn't provide the typesig so I stopped providing them too. HelloWorld was made before I realised this so I'm removing it to match the others.

**Comment by paf31** [comment]
Thanks!

File: exercises/hello-world/src/HelloWorld.purs (hunk 1 above)
Comment (paf31) [inline exercises/hello-world/src/HelloWorld.purs]: Sounds good.

**Pull request merged**